src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchViewModel.cs

```diff
@@ -501,10 +562,12 @@ internal PackageManagerSearchViewModel()
             ViewPackageDetailsCommand = new DelegateCommand<object>(ViewPackageDetails);
             ClearSearchTextBoxCommand = new DelegateCommand<object>(ClearSearchTextBox);
             ClearToastNotificationCommand = new DelegateCommand<object>(ClearToastNotification);
+            ClearFiltersCommand = new DelegateCommand<object>(ClearAllFilters);
             SearchText = string.Empty;
-            SortingKey = PackageSortingKey.LastUpdate;
+            SortingKey = PackageSortingKey.Votes;
```

Comment:
@hwahlstrom Would you explain this change again? After this change, how would a new package published surfaced to users? Should the author to write post on forum to encourage the trial given it would be more difficult for users to see the new packages

Reply:
This is per @Amoursol 's comment: "Downloads is a historical metric (Early packages typically = more downloads), so IMHO votes is better." However, I just realized that default sorting was last uploaded and not downloads. Users could apply the New filter to see new packages, but I am wondering if we should instead retain the default "last update" sorting.

Reply:
@hwahlstrom If I may, I feel we should retain the default "last update" sorting, that way newer packages are always on top of the default list view but not the most downloaded or voted ones..

Reply:
Yes, let's go with that.

Reply:
Reverted